### PR TITLE
Cleanup unused code

### PR DESCRIPTION
**Pull request by lkwg82**
### Purpose

come accross https://github.com/alecthomas/gometalinter and cleaned up some occurences

### Testing

clean and recompile

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
Not sure why you are touching vendored packages

**Comment by lkwg82** [comment]
fixed, was not my intention

**Comment by AudriusButkevicius** [comment]
@st-review merge

all: Clean up dead code

**Comment by st-review** [comment]
:clock1: Build status is `pending`. I'll wait to see if this becomes successful and then merge!

File: lib/model/rwfolder.go [hunk 1]
```diff
@@ -1492,16 +1492,6 @@ func (p *rwFolder) inConflict(current, replacement protocol.Vector) bool {
 	return false
 }
 
-func invalidateFolder(cfg *config.Configuration, folderID string, err error) {
```
Comment (calmh) [inline lib/model/rwfolder.go]: This is only accidentally dead, as it should be used from a couple of places. There's an issue on fixing that in the planned or v0.13 milestones.

**Comment by st-review** [comment]
:ok_hand: Merged as e87c1abd4e4391a450681b75be08f10ff9ce6869. Thanks, @lkwg82!

**Comment by calmh** [comment]
No worries though, we'll reinstate as appropriate

**Pull request closed without merging**